learning_journal/learning_journal/models/myentry.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    UnicodeText,
+    Date
+)
+
+from .meta import Base
+
+
+class MyEntry(Base):
+    __tablename__ = 'models'
+    id = Column(Integer, primary_key=True)
+    title = Column(UnicodeText)
+    body = Column(UnicodeText)
+    creation_date = Column(Date)
```

Comment:
This won't keep track of the time when an entry was written, but that should be okay.  Might be a bit problematic though if you write Tuesday's entry on Wednesday  morning before class and then write Wednesday's entry on Wednesday night.  I would consider updating this to a DateTime column type.